docs/examples/bicepconfig.json

```diff
@@ -0,0 +1,55 @@
+{

```

Comment:
Why include this file?

Reply:
should not be needed. We will eventually add the CLI command to generate a config file.

Reply:
This is currently THE sample config file visible to users